koku/sources/sources_error_message.py

```diff
@@ -68,12 +76,14 @@ def _extract_from_validation_error(self):
         if isinstance(self._error, ValidationError):
             err_dict = self._error.detail
             LOG.info(f"validation error: {self._error}. Validation detail {err_dict}")
-            err_key = list(err_dict.keys()).pop()
-            err_body = err_dict.get(err_key, []).pop()
-            if err_body:
-                err_msg = err_body.encode().decode("UTF-8")
-            else:
+            err_key = list(err_dict).pop()
+            try:
+                err_body = err_dict.get(err_key, []).pop()
+            except (TypeError, IndexError):
```

Comment:
The whole purpose of this `_extract_from_validation_error` is to pull the `ErrorDetail` out and ship that back to Sources. The `err_msg` we return here is displayed in the UI, so we cannot send back the full `err_body`.